assembl/static2/workspaces/draft-js-side-toolbar-plugin/src/components/Toolbar/index.js

```diff
@@ -41,6 +41,7 @@ class Toolbar extends React.Component<SideToolbarProps> {
   }
 
   componentDidMount() {
+    document.addEventListener('scroll', this.trackScrolling);
```

Comment:
We usually use `debounce` for scroll, otherwise setState may be called too often and you can get a react warning.
And you need to add a `window.removeEventListener` in `componentWillUnmount`